packages/server/src/app/api/notification/notification.controller.ts

```diff
@@ -0,0 +1,14 @@
+import { Controller, Post } from '@nestjs/common';
+
+import { CreateNotification } from '@pulp-fiction/models/notifications';
+import { NotificationsService } from '../../db/notifications/notifications.service';
+
+@Controller('notification')
+export class NotificationController {
+    constructor(private readonly notificationsService: NotificationsService) {}
```

Comment:
The controller is directly using the DB's `NotificationService`. I remember that most everywhere else we have an extra layer of indirection--an API-level `<Thing>Service` that then hooks into DB-level services if it needs to. Not really sure we need that extra layer here?